packages/assets/lib/external/network_config.js

```diff
@@ -0,0 +1,46 @@
+/* @license Copyright 2023 @polkadot-cloud/library authors & contributors
+SPDX-License-Identifier: GPL-3.0-only */
+export const polkadot_network_url =
+  "https://raw.githubusercontent.com/paritytech/polkadot_network_directory/master/chain_info/";
+
+export const networks = [
```

Comment:
Upper case camel for these consts?
```suggestion
export const POLKADOT_NETWORK_URL =
  "https://raw.githubusercontent.com/paritytech/polkadot_network_directory/master/chain_info/";

export const NETWORKS = [
```